tests/interfaces-network/task.yaml

```diff
@@ -0,0 +1,85 @@
+summary: |
+  The network interface allows a snap to access the network as a client.
+
+  A snap which defines the network plug must be shown in the interfaces list.
+  The plug must be autoconnected on install and, as usual, must be able to be
+  reconnected.
+
+  A snap declaring a plug on this interface must be able to access network services.
+
+environment:
+  SNAP_NAME: network-consumer
+  SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+  PLUG: network
```

Comment:
Done, this could allow us to share common bits between all the interface tests, but now it isn't useful at all